app/player/[id]/page.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
```

Comment:
the function is async and it doesn't seems to be any asynchronous operation being performed in this code , also since the app is simple and dosn't interact with any parent components or share any state with them , it makes more sense to user the PlayerInfo component directly to avoid rendering extra dom elements